### ISSUE TITLE
Black screen at respawn

### ISSUE THREAD

**Issue by pennilesswitch** [anegostudios/VintageStory-Issues#4694]
### Game Version

v1.20.0-rc4

### Platform

Windows

### Modded

Modded

### SP/MP

Multiplayer

### Description

playing multiplayer with family, if we die we sporadically respawn with black screen. You can move, but not interact with anything. I dies several times during a light temporal storm due to getting black screen. My daughter and husband got it as well. The only way to fix it is to close the game and restart. The only mods we are using are Fields and Plateaus and Plains and Valleys. This also happened with v. 1.19.

### How to reproduce

Play multiplayer.

### Screenshots

![20241214191216_1](https://github.com/user-attachments/assets/4da16388-08b5-4ef7-86ed-84b4f9a964fa)

### Logs

[server-main.log](https://github.com/user-attachments/files/18138337/server-main.log)

[client-main.log](https://github.com/user-attachments/files/18138338/client-main.log)

**Comment by Craluminum2413** [comment]
Have you tried to reinstall the game and update/downgrade/reinstall graphics drivers?

**Comment by rk43devr** [comment]
Any news about this issue?

I have this issue as well. 
Version: V1.20.0-rc.5
Mods: Auto Map Markers, Carry On, (Plains and Valleys but disabled)
W11, Ryzen 3900X, 1080TI

Screen stays black after respawning. It worked with the default spawn location. I used a Temp gear to change the spawn and that worked for a while. Now i have a black screen with every respawn.

**Comment by Craluminum2413** [comment]
@rk43devr Try rc.6

**Comment by rk43devr** [comment]
It's still happening here. 

Installed the update, updated the save file. 
Did a /kill in base and spawned in base without a black screen.
Died in a cave a long way from base without black screen.
Just died in the base because of drifters and it gave me a black screen.

When a black screen happens i still can move, have the UI and still have audio. It's just that the screen is black like the fade in after respawn isn't triggering.

After the respawn the client logs complain about : [Warning] OpenAL Error during stop of sound game:sounds/environment/wind.ogg: InvalidValue

**Comment by ninjaguy454** [comment]
I'd like to add onto this. 

I had this issue while playing on Steam OS running 1.20.1, I've updated it to 1.20.3, and haven't experienced it yet even after dying about a dozen times.

**Comment by pennilesswitch** [comment]
We still have it happening. It doesn't matter if it is vanilla or not. We
removed the game and redownloaded it. It still does it. Tried changing
screen size, loading distance, it just keeps happening. We also have
assets, usually stone rubble or banners not loading correctly. We also do
not see the food when put into the oven or on the table. It's just
invisible.

**Comment by TheRealNiceNix** [comment]
(Apologies for the duplicate bug I submitted a minute ago, thanks for closing it Craluminum2413.)
This is definitely the same issue I am experiencing.  Happens in both single-player and LAN multi-player.  Playing on Windows10 with mods: Better Firepit, Buzzwords, HUD Clock Patch, Cartographer, and Terra Prety.  I have only seen it since the 1.20.3 update.  Unfortunately I also installed the mods at the same time as the update so I can't isolate which caused the issue.  Previously played a lot of 1.16-1.19 vanilla and this bug did not happen then.

**Comment by neolooker** [comment]
i just got the same issue, what worked for me was to press F11 twice to switch from full screen to windowed and back, idk if its a feature of the game or not since i can do the same in some other games so maybe switching from full screen to windowed in the settings might work?

**Comment by PortalFan15** [comment]
Still happening in 1.21.1

**Comment by Iyonia** [comment]
It's happening to me, too! I'm playing completely unmodded, and it happens in both single player and LAN. I haven't played enough on a server over the internet to change anything. Closing and reloading the game gets rid of the issue, but then it happens again immediately upon the next death, ad infinitum.

**Comment by pennilesswitch** [comment]
We have found that if you hit the f5 button repeatedly, it will sometimes
fix the problem.

**Comment by Zuminap** [comment]
I don't know how the hell it works, but I solved the problem by turning off the second monitor

**Comment by BlitzRaus** [comment]
Same here, happens during combat scenarios.

**Comment by Niffen36** [comment]
Yep Its happening to me also now. latest update 1.22.2 or whatever.

Edit: Just to add, This now happens in combat, or mining. I've had good experience pressing F11 twice, prior to having this issue. If you wait until the issue occurs, it doesn't work, but if you do it when you first load into the game, it seems to be less likely to happen.

**Comment by AmonBune** [comment]
Hey, I made a mod that fixes this issue:

**Respawn Black Screen Fix**
[https://mods.vintagestory.at/show/mod/54860](https://mods.vintagestory.at/show/mod/54860)

The issue lies deeply inside the game engine itself which is not fully open sourced (yet?) by the devs. This mod is basically just patching the symptoms, but it works. I hope they will fix it in their own code someday.
